tests/test-runner.lua

```diff
@@ -143,7 +124,9 @@ local ok, err = pcall(a.run_blocking, function()
         log.info("Testing packages", packages)
 
         for __, pkg_path in ipairs(packages) do
-            local pkg = try(parse_package_spec(pkg_path))
+            -- Turns "packages/rust-analyzer/package.yaml" into "rust-analyzer"
+            local pkg_name = vim.fn.fnamemodify(pkg_path, ":h:t")
```

Comment:
For code readability, this could also be achieved with `vim.fs.basename(vim.fs.dirname(pkg_path))`